src/common/FPSTimer.h

```diff
@@ -13,6 +13,7 @@
 
 constexpr int NUM_HISTORY_DATA = 100;
 constexpr int NUM_FRAMES_TO_AVERAGE = 128;
+constexpr int FPSVALIDTHRESHOLD     = 5;

```

Comment:
May be too detailed, but is `FPS_VALID_THRESHOLD` more readable? Just like the above variables.

Reply:
Done